libqtile/core/manager.py

```diff
@@ -220,97 +230,37 @@ def conn(self):
     def selection(self):
         return self.core._selection
 
-    def setup_eventloop(self) -> None:
-        self._eventloop.add_signal_handler(signal.SIGINT, self.stop)
-        self._eventloop.add_signal_handler(signal.SIGTERM, self.stop)
-        self._eventloop.set_exception_handler(handle_exception)
-
-        logger.debug('Adding io watch')
-        self.core.setup_listener(self, self._eventloop)
-
-        self._stopped_event = asyncio.Event()
-
-        # This is a little strange. python-dbus internally depends on gobject,
-        # so gobject's threads need to be running, and a gobject "main loop
-        # thread" needs to be spawned, but we try to let it only interact with
-        # us via calls to asyncio's call_soon_threadsafe.
-        try:
-            # We import dbus here to thrown an ImportError if it isn't
-            # available. Since the only reason we're running this thread is
-            # because of dbus, if dbus isn't around there's no need to run
-            # this thread.
-            import dbus  # noqa
-            from gi.repository import GLib  # type: ignore
-
-            def gobject_thread():
-                ctx = GLib.main_context_default()
-                while not self._stopped_event.is_set():
-                    try:
-                        ctx.iteration(True)
-                    except Exception:
-                        logger.exception("got exception from gobject")
-            self._glib_loop = self.run_in_executor(gobject_thread)
-        except ImportError:
-            logger.warning("importing dbus/gobject failed, dbus will not work.")
-            self._glib_loop = None
-
     async def async_loop(self) -> None:
         """Run the event loop
 
         Finalizes the Qtile instance on exit.
         """
+        self.configure()
```

Comment:
Is this the only place `configure` is called? If so, can you privatise it?

Reply:
fair enough